api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -979,9 +979,9 @@ def pause(self, msg: Optional[str] = None) -> None:
 
         A human can resume the protocol in the Opentrons App or on the touchscreen.
 
-        This function returns immediately, but the next function call that
-        is blocked by a paused robot (anything that involves moving) will
-        not return until the protocol is resumed.
+        .. note::
+            In Python Protocol API versions 2.13 and earlier, the pause will only
```

Comment:
```suggestion
            In Python Protocol API version 2.13 and earlier, the pause will only
```
We use singular here because reasons 🧐